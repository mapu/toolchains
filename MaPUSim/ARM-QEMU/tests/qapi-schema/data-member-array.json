# valid array members
{ 'enum': 'abc', 'data': [ 'a', 'b', 'c' ] }
{ 'struct': 'def', 'data': { 'array': [ 'abc' ] } }
{ 'command': 'okay', 'data': { 'member1': [ 'int' ], 'member2': [ 'def' ] } }
