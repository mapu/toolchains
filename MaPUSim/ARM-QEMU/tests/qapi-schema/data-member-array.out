[OrderedDict([('enum', 'abc'), ('data', ['a', 'b', 'c'])]),
 OrderedDict([('struct', 'def'), ('data', OrderedDict([('array', ['abc'])]))]),
 OrderedDict([('command', 'okay'), ('data', OrderedDict([('member1', ['int']), ('member2', ['def'])]))])]
[{'enum_name': 'abc', 'enum_values': ['a', 'b', 'c']}]
[OrderedDict([('struct', 'def'), ('data', OrderedDict([('array', ['abc'])]))])]
